Update push docs for flag changes in Repo Integrity (1.12) (#103)
Oxen-AI/docs: getting-started/command-line/sync_remote.mdx

### Resume a Push

If a push is cancelled partway through, use `--missing-files` to resume and upload only the files the remote is missing.

```bash
oxen push --missing-files
```

> This is a fast repair — it uploads content the remote doesn't have — but it does not detect files that are present on the remote yet corrupted. To find and fix corruption too, use `--revalidate`.

Reviewer: ```suggestion
This is a fast repair. It uploads content the remote doesn't have, but it does not detect corrupted files. To find and fix corruption too, use `--revalidate`.
```

### Revalidate a Remote

> `--revalidate` runs a full integrity check on the remote: it re-hashes every stored file, deletes any whose contents no longer match their hash, then re-uploads everything that is now missing. It repairs corruption as well as absence, at the cost of scanning the remote's entire storage — so reach for it when you suspect corrupted content rather than a merely interrupted push.

Reviewer: ```suggestion
`--revalidate` runs a full integrity check on the remote. It removes corrupted files from the remote and then pushes all missing files. This is the right option to use if you suspect there are any corrupted files on the remote.
```
